### PR TITLE
Add a custom time parser

### PR DESCRIPTION
**Pull request by Brookke**
This handles unix time stamps as well as strings, once myradio is made to output consistent time formats this will be a lot more useful

### CODE REVIEW

**Comment by LordAro** [comment]
@Brookke you gonna fix this? :p

File: timeslot_test.go [hunk 1]
```diff
@@ -12,7 +12,10 @@ import (
 // It does NOT (yet) test the converse.
 func testCanEntryZero(t *testing.T) {
 	s := myradio.Show{}
-	if !s.IsZero() {
+	if !s.StartTime.IsZero() {
+		t.Error("zero show returns false for IsZero")
+	}
+	if !s.EndTime.IsZero() {
```
Comment (LordAro) [inline timeslot_test.go]: Hmm, combine into one if statement?

**Comment by Brookke** [comment]
@LordAro was mainly waiting till we had standardised myradios times but as that isn't happening for a while I guess this can be merged

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (Q-Mart) [inline .vscode/launch.json]: I think you've put this here by accident

Comment (Brookke) [inline .vscode/launch.json]: oops

**Comment by Brookke** [comment]
Need some help with solving this test error https://github.com/UniversityRadioYork/myradio-go/pull/37/files#diff-4a1f860ce0e234f8c45b46e0e8717605R40 @MattWindsor91 was this one of your tests

File: timeslot.go [hunk 3]
```diff
@@ -8,6 +8,28 @@ import (
 	"github.com/UniversityRadioYork/myradio-go/api"
 )
 
+// CustomTime is a custom time wrapper
```
Comment (MattWindsor91) [inline timeslot.go]: Seems like a bit of a non-indicative name.  `MyRadioTime` maybe?  IDK

**Pull request merged**